gtsfm/bundle/bundle_adjustment.py

```diff
@@ -131,6 +131,7 @@ def run(
         def get_metrics_from_sfm_data(sfm_data: GtsfmData, suffix: str) -> List[GtsfmMetric]:
             """Helper to get bundle adjustment metrics from a GtsfmData object with a suffix for metric names."""
             metrics = []
```

Comment:
I think number_images() is now kept fixed since the loader. I think we want the length of get valid camera indices?

Reply:
Yep. corrected it.